dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -374,6 +378,51 @@ enum ClickhousePolicyMode {
     Both,
 }
 
+#[derive(Debug, Args)]
+struct ChickenSwitchesArgs {
+    #[command(subcommand)]
+    command: ChickenSwitchesCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum ChickenSwitchesCommands {
+    /// Show a chicken switch at a given version
+    Show(ChickenSwitchesShowArgs),
+
+    /// Set the value of all chicken switches for the latest version
+    /// Values carry over from the latest version if unset on the CLI.
+    Set(ChickenSwitchesSetArgs),
+}
+
+#[derive(Debug, Clone, Args)]
+struct ChickenSwitchesSetArgs {
+    planner_enabled: bool,
+}
+
+#[derive(Debug, Clone, Copy, Args)]
+struct ChickenSwitchesShowArgs {
+    version: ChickenSwitchesVersionOrCurrent,
+}
+
+#[derive(Debug, Clone, Copy)]
+enum ChickenSwitchesVersionOrCurrent {
+    Current,
+    Version(u32),
+}
```

Comment:
nitpicky, feel free to ignore me: lately, for `omdb db` commands, we've been trying to get in the habit of putting new commands in their own files, [like this](https://github.com/oxidecomputer/omicron/tree/main/dev-tools/omdb/src/bin/omdb/db), mostly to minimize merge conflicts between branches that add new commands at the end of the file. but it also has the nice side benefit of meaning you don't need to prefix every args type with the entire name of the command, since they're in a module for that command...which may or may not actually matter to you, but i thought i'd mention it.

Reply:
Ah, I did not realize this. This is great! Thank you for pointing this out. I'm definitely going to do this.

Reply:
Done in [7531c64](https://github.com/oxidecomputer/omicron/pull/8449/commits/7531c641fdaae07fb8d9af5b657297cd6f117d1c)